mmaction/models/detectors/fast_rcnn.py

```diff
@@ -0,0 +1,249 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from mmdet.core import bbox2roi
+
+from mmaction.core.bbox import bbox2result, build_assigner, build_sampler
+from .. import builder
+from ..registry import DETECTORS
+from .base import BaseDetector
+
+
+@DETECTORS.register_module
+class FastRCNN(BaseDetector):
```

Comment:
Is it better to define a `TwoStageDetector` like `MMDet`, since many models on AVA are two-stage

Reply:
Do those models jointly train human detection & spatio-temporal action detection? It seems that all SOTA methods(including the winner of 2020 AVA challenge) use off-the-shelf detectors to get proposals. I think the current method is a reasonable baseline, We can extend this later if needed.